### DISCUSSION TITLE
arcade curriculum

### DISCUSSION THREAD

**Discussion by lverna** [community/community#37358] [Other Feature Feedback, Questions, & Ideas]
Hello

does anyone know where the MS arcade curriculum is located? https://arcade.makecode.com/ 

i went to https://aka.ms/TEALSintroCS and the link does not work.  [https://aka.ms/TEALSintroCS](url)

thanks!!

**Comment by ossanna16** [comment]
Hi @lverna,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Microsoft. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by lverna** [comment]
thanks Anna


would it be somewhere on this site?


https://docs.github.com/en


thanks


Luke Verna
he/him/his
Grades 6-8 Computer Technology Teacher
Upper Perkiomen Middle School
901 Montgomery Avenue
Pennsburg PA 18073 USA
267.313.4800 ext 6119
***@***.***


________________________________

**Reply by ossanna16** [reply]
@lverna Is this what you are looking for? https://arcade.makecode.com/courses

**Comment by lverna** [comment]
thanks!!!!!!!!!!!!!


at the bottom of that page


this link does not work for me


Download the course materials: https://aka.ms/TEALSintroCS


Luke Verna
he/him/his
Grades 6-8 Computer Technology Teacher
Upper Perkiomen Middle School
901 Montgomery Avenue
Pennsburg PA 18073 USA
267.313.4800 ext 6119
***@***.***


________________________________

**Reply by ossanna16** [reply]
Hi @lverna, unfortunately I'm unable to help with this and which is why I suggested asking the question in a Microsoft related forum instead. Our forum mostly deals with GitHub related questions.

**Comment by lverna** [comment]
ah, i see.  thanks


Luke Verna
he/him/his
Grades 6-8 Computer Technology Teacher
Upper Perkiomen Middle School
901 Montgomery Avenue
Pennsburg PA 18073 USA
267.313.4800 ext 6119
***@***.***


________________________________
